examples/scene-viewer/src/main.rs

```diff
@@ -74,12 +82,16 @@ fn load_skybox(renderer: &Renderer) -> Result<(), Box<dyn std::error::Error>> {
         .get_image_level_info(&file, 0, 0)
         .ok_or("skybox image missing")?;
 
+    let mips = transcoder.get_total_image_levels(&file, 0);
+
     let mut prepared = transcoder
         .prepare_transcoding(&file)
         .ok_or("could not prepare skybox transcoding")?;
     let mut image = Vec::with_capacity(image_info.total_blocks as usize * 16 * 6);
-    for i in 0..6 {
-        image.extend_from_slice(&prepared.transcode_image_level(i, 0, basis::TargetTextureFormat::Bc7Rgba)?);
+    for mip in 0..mips {
```

Comment:
Unfortunately, most tools lay out images the other way around `for layers { for mips { }}`, so in order to stay compatible with those tools (so you can just memcopy the data in), we need to remain consistent. This also means we can't copy all layers at once.

Reply:
I tried this with the included skybox in scene-viewer, and it worked correctly